lib/dropbox/client.rb

```diff
@@ -122,17 +122,12 @@ def get_thumbnail(path, format='jpeg', size='w64h64')
       return FileMetadata.new(resp), body
     end
 
-    def _list_folder(path, opts = nil)
-      args = { path: path }.merge(opts || {})
-      request('/files/list_folder', args)
-    end
-
     # Get the contents of a folder.
```

Comment:
I wasn't sure how you feel about this method `_list_folder`... It seems like the call to `request('/files/list_folder')` could be pulled out as a little helper since it's now used twice, but maybe not worth the overhead.
So in this second commit, i took that little refactor out.